README.md

```diff
@@ -25,13 +18,8 @@ Coverage.jl
 
 *Step 1:* Navigate to your test directory, and start julia like this:
 ```sh
-julia --code-coverage=user
-```
-or, if you're running Julia 0.4 or higher,
-```sh
-julia --code-coverage=user --inline=no
+julia --code-coverage=tracefile-%p.info --code-coverage=user
 ```
-(Turning off inlining gives substantially more accurate results, but may slow down your tests.)
```

Comment:
Is this not true anymore?

Reply:
Hasn't been true for years

Reply:
Huh, weird, I thought it was relevant until a few weeks ago, see https://github.com/JuliaLang/julia/issues/28192 . Anyway, glad to hear things are better now.

Reply:
It was fixed by #18196, the latest issue were just a regression from introduced the new SSAIR format and is re-fixed by #30251